gravityflow/gravityflow
# Add Multi-User Field

This PR adds support for a new Multi-User field. It works like the User field.

**Testing Instructions**
- Add a Multi-User field to a form
- Make the field editable in a User Input step
- Assign an Approval step to the field
- Check the workflow works as expected
- Check the assignee policy
- Check merge tags plus modifiers work as expected
- Check export works as expected
- Check the enhanced UI
- Try to break it!
